src/main/java/appeng/core/lib/features/BlockDefinition.java

```diff
@@ -85,4 +89,13 @@ public boolean isSameAs( Object other )
 		}
 	}
 
+	public IItemDefinition buildItemBlock( FeatureFactory factory){
```

Comment:
Definitions shouldn't have anything build releated in them. Because they are already defined... If that makes sense :D .